shared/haml/hamburger.haml

```diff
@@ -40,23 +45,32 @@
 
   show_teacher_options = "hide-always"
   show_student_options = "hide-always"
+  show_user_options = "hide-always"
   show_pegasus_options = "hide-always"
+  show_help_options = "show-always"
```

Comment:
If we're always going to explicitly set this below, then maybe good to start it as `"hide-always"` so that it's consistent with the others.